src/modules/qt/gps_parser.cpp

```diff
@@ -1200,6 +1226,8 @@ void qxml_parse_tcx(QXmlStreamReader &reader, gps_point_ll **gps_list, int *coun
                     crt_point.cad = reader.readElementText().toDouble();
                 } else if (reader.name() == QString("Temperature")) {
                     crt_point.atemp = reader.readElementText().toDouble();
+                } else if (reader.name() == QString("Power")) { // guessing
```

Comment:
A Shotcut forum user added a [sample tcx ](https://forum.shotcut.org/t/gps-text-video-filter/29278/34?u=daniel47) when requesting power to be added, it seems the Garmin schema uses this (a bit more complex) extension for storing power:

            <Extensions>
              <ns3:TPX>
                <ns3:Speed>0.0</ns3:Speed>
                <ns3:Watts>0</ns3:Watts>
              </ns3:TPX>
            </Extensions>

Reply:
FYI, if the namespace thing is too annoying to deal with or you don't care about the .gpx section I am fine with approoving this as is.

Reply:
Garmin is very popular, but we can merge this and then try to extend to tcx. I am getting the impression that we are not going to hear back from the contributor.